collector/database/chart.php

```diff
@@ -34,11 +34,24 @@
 
 $url = new moodle_url('/admin/tool/cloudmetrics/collector/database/chart.php');
 
+
```

Comment:
There should be at most 1 line of whitespace at a time. This file has several places with extra whitespace.